src/components/date-span/date-span.vue

```diff
@@ -102,9 +102,13 @@ export default {
       this.timeSpan.push(this.endDate);
     },
     timeExtent() {
-      this.extractStartEndDate();
+      this.startDate = this.timeExtent[0];
+      this.endDate = this.timeExtent[this.timeExtent.length - 1];
+      /* this.timeSpan = this.timeExtent; */
     },
     timeSpan() {
+      /* NOTE default timeSpan for now is set to days. I dont take into account the time value.
```

Comment:
Use TODO tag  instead of NOTE (will highlight and is more common to look for). Small thing, already happy with these kind of comments :)